doc/en/fcrontab.5.sgml

```diff
@@ -19,15 +19,16 @@ A copy of the license is included in gfdl.sgml.
     </refmeta>
     <refnamediv>
 	<refname>fcrontab</refname>
-	<refpurpose>tables for driving fcron</refpurpose>
+	<refpurpose>file format to define the schedule that fcron should adhere to</refpurpose>
```

Comment:
How about:
file format to define the jobs that fcron should run, and their schedules.

Reply:
yes, done